SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -410,18 +410,20 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	{
 		VirtualWorkerTranslator = MakeUnique<SpatialVirtualWorkerTranslator>();
 		VirtualWorkerTranslator->Init(this);
-		LoadBalanceEnforcer = NewObject<USpatialLoadBalanceEnforcer>();
-		LoadBalanceEnforcer->Init(Connection->GetWorkerId(), StaticComponentView, Sender, VirtualWorkerTranslator.Get());
 
 		// TODO: zoning - Move to AWorldSettings subclass [UNR-2386]
+		// TODO: Exit cleanly if SpatialSettings->LoadBalanceStrategy is not set.
```

Comment:
This TODO should be addressed, as UE4 crashes otherwise. At least a corresponding Jira ticket should be mentioned here.

Reply:
Used a default implementation instead and an error log.